src/components/BounceActivityContainer/index.stories.jsx

```diff
@@ -0,0 +1,75 @@
+import React from "react";
+import { storiesOf } from "@storybook/react";
+import StoryRouter from "storybook-react-router";
+import BounceRulesActivity from "./index";
+
+const log1 = {
+  bounce_action: "cypressDeleteTest",
```

Comment:
seems like there are some repeated object values in these log objects. we can store the reused ones in a variable and then spread them in each object so when someone is trying to see what the difference is between each log object, it's easier to see